src/main/scala/org/apache/mesos/chronos/scheduler/graph/JobGraph.scala

```diff
@@ -18,9 +21,9 @@ import scala.collection.mutable.{HashMap, ListBuffer, Map, SynchronizedMap}
 @ThreadSafe
 class JobGraph {
   val dag = new DirectedAcyclicGraph[String, DefaultEdge](classOf[DefaultEdge])
-  val edgeInvocationCount = Map[DefaultEdge, Long]()
+  val edgeInvocationCount = mutable.Map[DefaultEdge, Long]()
   private[this] val log = Logger.getLogger(getClass.getName)
-  private[this] val jobNameMapping = new HashMap[String, BaseJob] with SynchronizedMap[String, BaseJob]
+  private[this] val jobNameMapping: concurrent.Map[String, BaseJob] = new ConcurrentHashMap().asScala
```

Comment:
Do we still need the synchronized blocks in this class around jobNameMapping since it is a ConcurrentHashMap now?